CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+* [#154](https://github.com/withassociates/slices/pull/154): Fix issue where
```

Comment:
How about adding the `### Bug fixes` heading? Then the person doing the release doesn't have to figure that bit out (but they _will_ have to decide which part of version number to bump).